README.md

```diff
@@ -11,49 +11,378 @@
 
 By [Fauna, Inc](http://faunadb.com).
 
-## Inroduction
 
-## Usage
+This repository contains FaunaDB driver for Swift language. It provides high level abstractions that allows us to work with fauna DB efficiently and without the need to deal with rest messages, networking errors, data encoding, decoding and so on.
 
-```swift
-import FaunaDB
-..
-.
-```
+> working on another language? Take a look at our github account, we provide Fauna DB drivers for many popular languages and many others are coming. Don't hesitate to contact us if the language you are working on is not supported yet. Notice you can always use the [REST API](https://faunadb.com/documentation/rest) directly.
 
 ## Requirements
 
 * iOS 8.0+
 * Xcode 7.3.1+
+* Swift 2.2
 
-## Getting involved
+## Getting started
 
-* If you **want to contribute** please feel free to **submit pull requests**.
-* If you **have a feature request** please **open an issue**.
-* If you **found a bug** or **need help** please **check older issues and [FAQ](#faq) before submitting an issue.**.
+1. Install the driver first. We can install the driver either by using [Carthage](https://github.com/Carthage/Carthage), [CocoaPods](https://cocoapods.org/) or manually as Embedded Framework. Go to [Installation](#installation) section to know how.
 
-Before contribute check the [CONTRIBUTING](https://github.com/faunadb/faunadb-swift/blob/master/CONTRIBUTING.md) file for more info.
+2. If you haven’t already, [sign up for an account](https://faunadb.com/#signup-modal) to start using FaunaDB.
 
-If you use **FaunaDB** in your app We would love to hear about it! Drop us a line on [twitter](https://twitter.com/faunadb).
+3. From here, Fauna DB [tutorial](https://faunadb.com/tutorials) will walk you through step by step how to get started with Fauna DB.
+
+#### Basic Usage
 
-## Examples
+```swift
+import Foundation
+import Result
+import FaunaDB
 
-Follow these 3 steps to run Example project: Clone FaunaDB repository, open FaunaDB workspace and run the *Example* project.
+// create a fauna client
+// Logger will shows in the Xcode console the curl representation of each message sent to FaunaDB, it also shows the fauna response data. This is useful during app development.
 
-You can also experiment and learn with the *FaunaDB Playground* which is contained in *FaunaDB.workspace*.
+let secret = <YOUR_CLIENT_SECRET>
+let dbName = <YOUR_DB_NAME>
+var client = Client(secret: secret, observers: [Logger()])
 
-## Installation
+func setUpSchema(callback: (Result<Value, Error> -> ()){
```

Comment:
Remove schema setup. We should have a simpler example, like matching an index, or creating an instance and parsing its data.